### PR TITLE
ui: update readme with `make watch` clarification

### PR DESCRIPTION
**Pull request by mayank2k16**
Updated the README.md making it more clear about "make watch" command.

### CODE REVIEW

**Comment by vilterp** [comment]
Hey Mayank, thanks for submitting this.  `make ui-watch` works if you are in the root directory of Cockroach, and `make watch` works if you are in `pkg/ui`. So, `make watch` is not deprecated. I think this readme should make that more clear, though. (A couple of other commands work this way, e.g. `make ui-lint` and `make ui-test`).

Mind updating the readme to say the above?

Also, please format your commit message like this:

```
ui: update readme about `make watch` commands

Release note: None
```

**Comment by mayank2k16** [comment]
Hey @vilterp i updated the README.md making it more clear about "make watch" command.

File: pkg/ui/README.md [hunk 1]
```diff
@@ -41,18 +41,26 @@ We've created a simple NodeJS proxy to accomplish this. This server serves all
 requests for web resources (JavaScript, HTML, CSS) out of the code in this
 directory, while proxying all API requests to the specified CockroachDB node.
 
-To use this proxy, run
+To use this proxy, in your root directory run
```
Comment (vilterp) [inline pkg/ui/README.md]:
```suggestion
To use this proxy, in Cockroach's root directory run:
```

File: pkg/ui/README.md [hunk 2]
```diff
@@ -41,18 +41,26 @@ We've created a simple NodeJS proxy to accomplish this. This server serves all
...
+```shell
+$ make ui-watch TARGET=<target-cluster-http-uri>
+```
 
+or, in your pkg/ui directory run
```
Comment (vilterp) [inline pkg/ui/README.md]:
```suggestion
or, in `pkg/ui` run:
```

File: pkg/ui/README.md [hunk 3]
```diff
@@ -41,18 +41,26 @@ We've created a simple NodeJS proxy to accomplish this. This server serves all
...
 ```shell
 $ make watch TARGET=<target-cluster-http-uri>
 ```
 
 then navigate to `http://localhost:3000` to access the UI.
 
-To proxy to a cluster started up in secure mode, use:
+To proxy to a cluster started up in secure mode, use this in your root directory:
```
Comment (vilterp) [inline pkg/ui/README.md]:
```suggestion
To proxy to a cluster started up in secure mode, in Cockroach's root directory run:
```

File: pkg/ui/README.md [hunk 4]
```diff
@@ -41,18 +41,26 @@ We've created a simple NodeJS proxy to accomplish this. This server serves all
...
+```shell
+$ make ui-watch-secure TARGET=<target-cluster-https-uri>
+```
 
+or, use this in pkg/ui directory
```
Comment (vilterp) [inline pkg/ui/README.md]:
```suggestion
or, in `pkg/ui` run:
```

**Review by vilterp — COMMENTED** [review]
Thanks. Made a couple suggestions; please squash them into the original commit. Also, is accepting the CLA not working? We'll need that before merging.

**Comment by mayank2k16** [comment]
@vilterp Thanks for your suggestions. They are more precise. I updated them.

**Comment by vilterp** [comment]
Hm, looks like the updates got lost, maybe when you were squashing. Try again?

**Comment by mayank2k16** [comment]
@vilterp Just squashed them. Now its updated.

File: pkg/ui/README.md [hunk 5]
```diff
@@ -41,18 +41,26 @@ We've created a simple NodeJS proxy to accomplish this. This server serves all
...
-To proxy to a cluster started up in secure mode, use:
+To proxy to a cluster started up in secure mode, in Cockroach's root directory run:
+```shell
+$ make ui-watch-secure TARGET=<target-cluster-https-uri>
+```
 
+or, in `pkg/ui` run:
 ```shell
-$ make watch-secure TARGET=<target-cluster-https-uri>
+make watch-secure TARGET=<target-cluster-https-uri>
```
Comment (vilterp) [inline pkg/ui/README.md]: Tiny nit:
```suggestion
make watch-secure TARGET=<target-cluster-https-uri>
```

**Review by vilterp — APPROVED** [review]
Looks good pending last tiny nit.

Regarding the CLA, seems like you need to hook up your GitHub account; seems like it can't connect your commits to your account right now.

**Comment by couchand** [comment]
LGTM.  I've updated the PR description so that when bors merges it the summary will be right, so once we get cla-bot happy we can merge this.  Thanks!

**Pull request merged**